tests/Admin/AdminHelperTest.php

```diff
@@ -143,32 +143,28 @@ public function testAppendFormFieldElement(): void
 
         $associationMapping = [
             'fieldName' => 'bar',
-            'targetEntity' => Foo::class,
+            'targetEntity' => Bar::class,
```

Comment:
Why do you need to change this ?
I mean was the test wrong or are we loosing a feature ?

Reply:
It didn't make the test fail, but it's just logic that the target for the "bar" field is the Bar entity. Thought it was a good idea to make it relevant for comprehension.

Reply:
Can you revert this change to help the review (in order to not have this change in the same commit than the fix)

Best would be
- The commit with the test failing
- The commit with the fix, and green tests
- Then we can do a third commit with this change

This way will be sure that your code change fix the bug and doesn't introduce any others bugs :)

Reply:
Nice, it works fine.

Reply:
You can re-add the change if you want to.